src/pages/evm/wallet/SendPage.vue

```diff
@@ -457,6 +474,21 @@ export default defineComponent({
     &__title {
         font-size: 2.4rem;
         font-weight: 600;
+        margin: 0px;
+    }
+
+    &__title-from {
+        font-size: 12px;
+        font-weight: 400;
+        margin: -7px 0px 0px 0px;
+    }
+
+    &__title-address {
+        font-size: 16px;
+        .c-user-info__address {
+            margin-right: 0px;
+            font-weight: 400;
+        }
```

Comment:
can you move this alternate styling to the `UserInfo` component and control it with a prop? we should avoid style shadowing like this

Reply:
done